src/lisp/regression-tests/system-construction.lisp

```diff
@@ -21,13 +21,13 @@
     (cmp::cfp-output-extension :fasl)
     (pathname-type (compile-file-pathname "test.lisp")))))
 
-(test-true compile-file-pathname-3a
+#+(or)(test-true compile-file-pathname-3a
```

Comment:
if we no longer support bitcode output we should just axe this test code.

Reply:
Axed.